<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtocolDialog</class>
 <widget class="QDialog" name="ProtocolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>协议信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f5f7fa;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="styleSheet">
      <string notr="true">
      font-size: 16px;
      font-weight: bold;
      color: #333333;
      </string>
     </property>
     <property name="text">
      <string>协议信息</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="formContainer" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 12px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="horizontalSpacing">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelName">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>协议名称：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineName">
        <property name="styleSheet">
         <string notr="true">
          QLineEdit {
              padding: 8px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 13px;
              background-color: white;
              color: #333333;
          }
          QLineEdit:focus {
              border: 1px solid #4a90e2;
              box-shadow: 0 0 0 2px rgba(74,144,226,0.2);
          }
         </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelType">
        <property name="styleSheet">
         <string notr="true">color: black;</string>
        </property>
        <property name="text">
         <string>协议类型：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboType">
        <property name="styleSheet">
         <string notr="true">
         QComboBox {
             padding: 8px 12px;
             border: 1px solid #d0d0d0;
             border-radius: 4px;
             font-size: 13px;
             background-color: white;
             color: black;  /* 主文本颜色改为黑色 */
         }
         QComboBox::drop-down {
             width: 20px;
             border: none;
         }
         QComboBox QAbstractItemView {
             color: black;  /* 下拉列表文本颜色 */
             background-color: white;  /* 下拉列表背景 */
             selection-background-color: #3a6ea5;  /* 选中项背景 */
             selection-color: white;  /* 选中项文字颜色 */
         }
         </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDesc">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>协议描述：</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QTextEdit" name="textDescription">
        <property name="styleSheet">
         <string notr="true">
          QTextEdit {
              padding: 8px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              font-size: 13px;
              background-color: white;
              color: #333333;
          }
          QTextEdit:focus {
              border: 1px solid #4a90e2;
              box-shadow: 0 0 0 2px rgba(74,144,226,0.2);
          }
         </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonContainer" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOK">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #27ae60;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #2ecc71;
          }
          QPushButton:pressed {
              background-color: #219955;
          }
         </string>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/ok.png</normaloff>:/icons/ok.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #95a5a6;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #7f8c8d;
          }
          QPushButton:pressed {
              background-color: #6c7a7d;
          }
         </string>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>ProtocolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ProtocolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
